normandy/studies/models.py

```diff
@@ -18,9 +18,13 @@
 SIGNING_FILES = {"META-INF/mozilla.rsa", "META-INF/mozilla.sf", "META-INF/manifest.mf"}
 
 
+def get_extension_filename(instance, *args):
+    return f"extensions/{instance.extension_id}-{instance.version}-signed.xpi"
```

Comment:
I don't think we should do this. I think the original filename is useful information we should preserve. If we really want to standardize names, I think we should at least store the original in the API.